theme-dev/README.md

```diff
@@ -220,26 +222,31 @@ These are provided by the Plugin (or Xposed module). They theme many things at o
 
 | String          | Purpose                 |
 | ---------------- |:-----------------------:|
-| drawable_button_grey | unread messages button |
+| drawable_button_grey | Unread messages button |
 | drawable_overlay_channels_selected_dark | Selected channel color in channel list for dark mode |
 | drawable_overlay_channels_pressed_dark | Pressed channel color in channel list for dark mode | 
-| item_background_material | mostly used for the top bar(where the name of the plugin, version and author name is written) |
-| design_bottom_navigation_item_background | is used mostly for the bottom part of the plugin page(where description is, uninstall and settings) also themes search box in plugins page |
-| drawable_button_red | the red NEW↑ in the guild list when you get a ping in a guild |
+| item_background_material | Mostly used for the top bar (where the name of the plugin, version and author name is written) |
+| design_bottom_navigation_item_background | Is mainly for the bottom part of the plugin page (where description is, uninstall and settings). Also themes' search box in plugins page |
```

Comment:
Was the ' supposed to be there?

Reply:
well yes, but I can remove it if it looks misplaced